tests/unit/swagger/definitionsGeneration/definitions.spec.ts

```diff
@@ -318,6 +318,29 @@ describe('Definition generation', () => {
                     expect(propertySchema).to.not.haveOwnProperty('additionalProperties', `for property ${propertyName}`);
                     expect(propertySchema['x-nullable']).to.eq(true, `for property ${propertyName}[x-nullable]`);
                 },
+                objLiteral: (propertyName, propertySchema) => {
+                    expect(propertySchema).to.deep.include({
```

Comment:
Can you use deep.equal instead?

Reply:
Prefer not to. I'm not setting any of default, description or format (which I understand to be not allowed in this context). Therefore, adding these 3 undefined props add a lot of unnecessary noise imho.

Reply:
I understand that it adds noise, but it could potentially prevent a future bug if it caches something. I always prefer quality control over readability.

Reply:
https://github.com/lukeautry/tsoa/pull/415/commits/da71a8960e79c534cf951deec603fe3e155a2aa7